admiral/cmd/admiral/cmd/root.go

```diff
@@ -103,6 +104,8 @@ func GetRootCmd(args []string) *cobra.Command {
 		"The label value, on a namespace, which tells Istio to perform sidecar injection")
 	rootCmd.PersistentFlags().StringVar(&params.HostnameSuffix, "hostname_suffix", "global",
 		"The hostname suffix to customize the cname generated by admiral. Default suffix value will be \"global\"")
+	rootCmd.PersistentFlags().StringVar(&params.PreviewHostnamePrefix, "preview_hostname_prefix", "preview",
```

Comment:
I dont think we need to make it configurable, lets just use `preview` the standard name for bluegreen strategy.